extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-mysql/src/main/java/org/apache/guacamole/auth/mysql/MySQLAuthenticationProviderModule.java

```diff
@@ -79,6 +84,37 @@ public MySQLAuthenticationProviderModule(MySQLEnvironment environment)
         // Allow use of multiple statements within a single query
         driverProperties.setProperty("allowMultiQueries", "true");
         
+        // Set the SSL mode to use when conncting
+        MySQLSSLMode sslMode = environment.getMySQLSSLMode();
+        driverProperties.setProperty("sslMode", sslMode.toString());
```

Comment:
With `MySQLSSLMode` now refactored to rely on `@PropertyValue`, this may not return the value you expect. The `MySQLSSLMode` enum should expose a means of retrieving the value required by the JDBC driver.